extract-metadata.py

```diff
@@ -20,18 +20,44 @@ def extract_md(filename):
 
 def extract_xml(filename):
     parser = xml.sax.make_parser()
-    handler = PIHandler()
+    handler = MdHandler()
```

Comment:
Yeah, that name is unfortunate.  Perhaps we can avoid "md" as a shorthand for "metadata" when it can (and does) already mean "markdown".

```suggestion
    handler = XmlHandler()
```